stable/nfs-server-provisioner/templates/statefulset.yaml

```diff
@@ -26,6 +26,10 @@ spec:
       # NOTE: This is 10 seconds longer than the default nfs-provisioner --grace-period value of 90sec
       terminationGracePeriodSeconds: 100
       serviceAccountName: {{ if .Values.rbac.create }}{{ template "nfs-provisioner.fullname" . }}{{ else }}{{ .Values.rbac.serviceAccountName | quote }}{{ end }}
+      {{- if .Values.imagePullSecrets }}
+      imagePullSecrets:
+{{ toYaml .Values.imagePullSecrets | indent 8 }}
```

Comment:
```suggestion
        {{- toYaml .Values.imagePullSecrets | nindent 8 }}
```

Reply:
solved in https://github.com/helm/charts/pull/18253/commits/552cd97840e943534c83dbdb0c717d08d8915de9